Sources/ArcGISToolkit/Components/Offline/OfflineMapInfo.swift

```diff
@@ -0,0 +1,139 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import Foundation
+import OSLog
+import UIKit.UIImage
+
+/// Information for an online map that has been taken offline.
+public struct OfflineMapInfo: Sendable {
```

Comment:
No longer codable because of the thumbnail. But that's easily solved by creating an info state object "CodableInfo". See below.